docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -2022,7 +2022,7 @@ see the expected update times in the v1.5.0 release.
 
 - Added additional logging around bubbling votes.
 
-## V1.5.1-eth_call ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.5.1-eth_call))
+## `V1.5.1-eth_call` ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.5.1-eth_call))
```

Comment:
Why are we adding the "`" back here? None of the other titles include this

Reply:
(happy to do this in a separate PR if that's the issue...)

Reply:
That is one way to ignore the heading capitalization rule.